src/modules/SimpleExecutionValidator.sol

```diff
@@ -29,6 +29,8 @@ contract SimpleExecutionValidator is IValidator {
 
     function isModuleType(uint256 typeID) external view override returns (bool) { }
 
+    function getModuleTypes() external view returns (uint256) { }
```

Comment:
@zeroknots it could also be useful to add the ModuleType lib here: https://github.com/zeroknots/gastest-moduletype/blob/main/src/ModuleType.sol

Reply:
done in:

https://github.com/erc7579/uMSA/pull/8/commits/fa414ea5d6f674b862f0127249fad88c81600e84